lib/Wildcard.js

```diff
@@ -0,0 +1,28 @@
+var Term = require('n3').DataFactory.internal.Term;
+
+// Wildcard constructor
+class Wildcard extends Term {
+  constructor() {
+    super('');
+    return WILDCARD || this;
+  }
+
+  get termType() {
+    return 'Wildcard';
+  }
+
+  equals(other) {
+    return other && (this.termType === other.termType);
+  }
+}
+
+Object.defineProperty(Wildcard.prototype, 'value', {
```

Comment:
You could also do this as well for `termType`. Was there any particular reason for handling these differently?

Reply:
Nothing in particular, will change!